packages/neos-ui/src/Containers/LeftSideBar/NodeTree/style.module.css

```diff
@@ -6,7 +6,22 @@
     background: var(--colors-ContrastDarker);
     border-bottom: 1px solid var(--colors-ContrastDark);
     border-right: 1px solid var(--colors-ContrastDark);
+    position: relative;
 }
 .loader {
     margin: var(--spacing-Quarter);
 }
+.collapseAll {
+    position: absolute;
+    right: 0;
+    top: 0;
+    opacity: .5;
+    cursor: pointer;
+    z-index: 5;
+    padding: 8px;
+    font-size: 1.4em;
+
+    &:hover > svg {
```

Comment:
I think `opacity: 1` would also be great on hover.

Reply:
Added!